winterfell/webapp/src/styles/claimSelectForms.styl

```diff
@@ -1,97 +1,187 @@
 @import "standards.styl"
 
-iconWrapperSize = 36px
-iconSize = 24px
-
 #select-forms-view
-  & .header-wrapper
-    height: megaSpacing
-    line-height: megaSpacing
-    text-align: center
-    background-color: colorPrimaryBlue
-    color: colorWhite
-
-  & .footer-wrapper
-    height: (2 * megaSpacing)
-    padding-top: smallSpacing
-    background-color: colorPrimaryDarkBlue
-
-    & .vfi-button-wrapper
-      width: maxLayoutWidthPx
-      max-width: maxLayoutWidthPct
-      margin: largeSpacing auto
-
-    & .vfi-cancel-btn
+  width: 100%
+  background-color: colorWhite
+
+#select-forms-wrapper
+  width: maxLayoutWidthPct
+  max-width: maxLayoutWidthPx
+  margin: 0 auto
+  padding: largeSpacing 0
+
+  & h3
+    margin-bottom: mediumSpacing
+    color: colorPrimaryBlue
+
+  & h4
+    display: inline-block
+    line-height: 40px
```

Comment:
nervous about these hardcoded text sizes, cant we just use `em` or something more mobile friendly?